File: SourceCode/Cassandra_Python/keyspace_operations/Alter_Keyspace.py
'''
Created on 18 Jul 2017

@author: spotnuru
'''
from cassandra.cluster import Cluster
cluster = Cluster(['127.0.0.1'])
session = cluster.connect()
session.execute("ALTER KEYSPACE task1 WITH REPLICATION = {'class' : 'NetworkTopologyStrategy', 'datacenter' : 4 };")
print("Keyspace Altered")

File: SourceCode/Cassandra_Java/cassandra_keyspace/Alter_KeySpace.java
package cassandra_keyspace;

import com.datastax.driver.core.Cluster;
import com.datastax.driver.core.Session;

public class Alter_KeySpace {
   public static void main(String args[]){

      //Query
      String query = "ALTER KEYSPACE xyz WITH replication " + "= {'class':'NetworkTopologyStrategy', 'datacenter1':3}"
         + "AND DURABLE_WRITES = false;";

      //Creating Cluster object
      Cluster cluster = Cluster.builder().addContactPoint("127.0.0.1").build();
   
      //Creating Session object
      Session session = cluster.connect();
 
      //Executing the query
      session.execute(query);
 
      System.out.println("Keyspace altered");
   }
}
